  INFO root [0x000018d0] [2018-07-22 11:06:36,054] - 
  INFO root [0x000018d0] [2018-07-22 11:06:36,101] - -----------------------------------------------------------------------------------------


  INFO root [0x000018d0] [2018-07-22 11:06:39,813] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x000018d0] [2018-07-22 11:06:39,814] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x000018d0] [2018-07-22 11:06:39,814] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x000018d0] [2018-07-22 11:06:40,633] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x000018d0] [2018-07-22 11:06:40,704] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x000018d0] [2018-07-22 11:06:40,709] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50083 )
  INFO root [0x000018d0] [2018-07-22 11:06:40,710] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50083, SOMAXCONN=128.
  INFO root [0x000018d0] [2018-07-22 11:06:40,711] - server-configs:
  INFO root [0x000018d0] [2018-07-22 11:06:40,711] - 	gameUpdateHertz : 10
  INFO root [0x000018d0] [2018-07-22 11:06:40,711] - 	entryScriptFile : kbemain
  INFO root [0x000018d0] [2018-07-22 11:06:40,712] - 	internalAddr : 192.168.0.106:50083
  INFO root [0x000018d0] [2018-07-22 11:06:40,712] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x000018d0] [2018-07-22 11:06:40,713] - 	componentID : 7001
  INFO root [0x000018d0] [2018-07-22 11:06:42,846] - Components::findLogger: found logger, addr:192.168.0.106:50074
  INFO root [0x000018d0] [2018-07-22 11:06:42,847] - Components::addComponent: reset orderLog, uid=6362!
  INFO root [0x000018d0] [2018-07-22 11:06:42,848] - Components::addComponent[logger], uid=6362, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x000018d0] [2018-07-22 11:06:43,255] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 6362. PID: 6096 ----
  INFO root [0x000018d0] [2018-07-22 11:06:43,256] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x000018d0] [2018-07-22 11:06:43,298] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x000018d0] [2018-07-22 11:06:43,352] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x000018d0] [2018-07-22 11:06:49,617] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000018d0] [2018-07-22 11:06:49,874] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x000018d0] [2018-07-22 11:06:50,140] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x000018d0] [2018-07-22 11:06:50,885] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x000018d0] [2018-07-22 11:06:51,005] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x000018d0] [2018-07-22 11:06:51,055] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x000018d0] [2018-07-22 11:06:51,103] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x000018d0] [2018-07-22 11:06:51,742] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x000018d0] [2018-07-22 11:06:52,857] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x000018d0] [2018-07-22 11:06:52,858] - Script::install(): KBExtra is successfully!
 DEBUG root [0x000018d0] [2018-07-22 11:06:53,975] - loaded entity-script:Account(1).
 DEBUG root [0x000018d0] [2018-07-22 11:06:53,976] - loaded entity-script:CheckName(2).
 DEBUG root [0x000018d0] [2018-07-22 11:06:54,067] - loaded entity-script:Halls(3).
 DEBUG root [0x000018d0] [2018-07-22 11:06:54,154] - loaded entity-script:Room(4).
  INFO root [0x000018d0] [2018-07-22 11:06:54,154] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x000018d0] [2018-07-22 11:06:54,280] - TelnetServer server is running on port 40000
  INFO root [0x000018d0] [2018-07-22 11:06:54,693] - ---- baseapp is running ----
 DEBUG root [0x000018d0] [2018-07-22 11:06:54,733] - Components::process(): Request for the process of identity...
  WARN root [0x000018d0] [2018-07-22 11:07:01,254] - Forwarding logs to logger[192.168.0.106:50074/0/0/0]...
 ERROR root [0x000018d0] [2018-07-22 11:09:12,307] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x000018d0] [2018-07-22 11:09:12,310] - The following logs sent to logger failed:
  WARN root [0x000018d0] [2018-07-22 11:09:12,310] - ==> [2018-07-22 11:09:12 306] TCPPacketReceiver::processPendingEvents(192.168.0.106:50074): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000018d0] [2018-07-22 11:09:12,310] - ==> [2018-07-22 11:09:12 307] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x000018d0] [2018-07-22 11:09:12,355] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50077): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x000018d0] [2018-07-22 11:09:12,356] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x000018d0] [2018-07-22 11:09:12,378] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x000018d0] [2018-07-22 11:09:12,378] - onBaseAppShutDown: state=0
  INFO root [0x000018d0] [2018-07-22 11:09:12,378] - Shutdowner::onShutdown: shutting down
S_INFO root [0x000018d0] [2018-07-22 11:09:12,379] - onBaseAppShutDown: state=1
  INFO root [0x000018d0] [2018-07-22 11:09:12,379] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x000018d0] [2018-07-22 11:09:12,379] - onBaseAppShutDown: state=2
 ERROR root [0x000018d0] [2018-07-22 11:09:12,487] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x000018d0] [2018-07-22 11:09:12,487] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,488] - Account::~Account(): 2002
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,488] - Halls::~Halls(): 2001
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,518] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,518] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,519] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,519] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,519] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,519] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,519] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x000018d0] [2018-07-22 11:09:12,519] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001dec] [2018-07-22 11:09:13,509] - 
  INFO root [0x00001dec] [2018-07-22 11:09:13,510] - -----------------------------------------------------------------------------------------


  INFO root [0x00001dec] [2018-07-22 11:09:13,510] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001dec] [2018-07-22 11:09:13,510] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001dec] [2018-07-22 11:09:13,510] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001dec] [2018-07-22 11:09:13,513] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00001dec] [2018-07-22 11:09:13,513] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50126 )
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50126, SOMAXCONN=128.
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - server-configs:
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - 	gameUpdateHertz : 10
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - 	entryScriptFile : kbemain
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - 	internalAddr : 192.168.0.106:50126
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00001dec] [2018-07-22 11:09:13,516] - 	componentID : 7001
  INFO root [0x00001dec] [2018-07-22 11:09:15,200] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 7006. PID: 6512 ----
  INFO root [0x00001dec] [2018-07-22 11:09:15,200] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001dec] [2018-07-22 11:09:15,201] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001dec] [2018-07-22 11:09:15,201] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,327] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,328] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,328] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,331] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,331] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,331] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,332] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,365] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001dec] [2018-07-22 11:09:15,399] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001dec] [2018-07-22 11:09:15,400] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,417] - loaded entity-script:Account(1).
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,418] - loaded entity-script:CheckName(2).
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,425] - loaded entity-script:Halls(3).
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,426] - loaded entity-script:Room(4).
  INFO root [0x00001dec] [2018-07-22 11:09:15,426] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001dec] [2018-07-22 11:09:15,426] - TelnetServer server is running on port 40000
  INFO root [0x00001dec] [2018-07-22 11:09:15,712] - ---- baseapp is running ----
 DEBUG root [0x00001dec] [2018-07-22 11:09:15,712] - Components::process(): Request for the process of identity...
  WARN root [0x00001dec] [2018-07-22 11:09:21,810] - EntityApp::checkTickPeriod: tick took 6096% (6.10 seconds)!
  INFO root [0x00001dec] [2018-07-22 11:09:22,912] - Components::findComponents: find logger(1)...
  INFO root [0x00001dec] [2018-07-22 11:09:22,985] - Components::findComponents: found logger, addr:192.168.0.106:50122
  INFO root [0x00001dec] [2018-07-22 11:09:22,986] - Components::addComponent: reset orderLog, uid=7006!
  INFO root [0x00001dec] [2018-07-22 11:09:22,986] - Components::addComponent[logger], uid=7006, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001dec] [2018-07-22 11:09:22,987] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001dec] [2018-07-22 11:09:23,089] - Components::findComponents: found dbmgr, addr:192.168.0.106:50119
  INFO root [0x00001dec] [2018-07-22 11:09:23,090] - Components::addComponent: reset orderLog, uid=7006!
  INFO root [0x00001dec] [2018-07-22 11:09:23,090] - Components::addComponent[dbmgr], uid=7006, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001dec] [2018-07-22 11:09:23,091] - Components::findComponents: found dbmgr, addr:192.168.0.106:50119
  WARN root [0x00001dec] [2018-07-22 11:09:24,091] - Forwarding logs to logger[192.168.0.106:50122/0/0/0]...
 ERROR root [0x00001dec] [2018-07-22 11:16:52,659] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00001dec] [2018-07-22 11:16:52,661] - The following logs sent to logger failed:
  WARN root [0x00001dec] [2018-07-22 11:16:52,661] - ==> [2018-07-22 11:16:52 656] TCPPacketReceiver::processPendingEvents(192.168.0.106:50122): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001dec] [2018-07-22 11:16:52,661] - ==> [2018-07-22 11:16:52 660] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00001dec] [2018-07-22 11:16:52,708] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50119): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00001dec] [2018-07-22 11:16:52,710] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00001dec] [2018-07-22 11:16:52,713] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00001dec] [2018-07-22 11:16:52,714] - onBaseAppShutDown: state=0
  INFO root [0x00001dec] [2018-07-22 11:16:52,714] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00001dec] [2018-07-22 11:16:52,714] - onBaseAppShutDown: state=1
  INFO root [0x00001dec] [2018-07-22 11:16:52,714] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00001dec] [2018-07-22 11:16:52,714] - onBaseAppShutDown: state=2
 ERROR root [0x00001dec] [2018-07-22 11:16:52,722] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00001dec] [2018-07-22 11:16:52,723] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00001dec] [2018-07-22 11:16:52,723] - Account::onCellWriteToDBCompleted(2004): not found dbmgr!
 ERROR root [0x00001dec] [2018-07-22 11:16:52,724] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,724] - Account::~Account(): 2002
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,724] - Account::~Account(): 2004
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,724] - Room::~Room(): 2003
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,725] - Halls::~Halls(): 2001
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,725] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,726] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,726] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,726] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,726] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,726] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,726] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,726] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00001dec] [2018-07-22 11:16:52,735] - Script::uninstall(): is successfully!
 DEBUG root [0x00001dec] [2018-07-22 11:16:52,736] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00002448] [2018-07-22 11:16:53,838] - 
  INFO root [0x00002448] [2018-07-22 11:16:53,838] - -----------------------------------------------------------------------------------------


  INFO root [0x00002448] [2018-07-22 11:16:53,839] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00002448] [2018-07-22 11:16:53,839] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00002448] [2018-07-22 11:16:53,839] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00002448] [2018-07-22 11:16:53,842] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00002448] [2018-07-22 11:16:53,842] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00002448] [2018-07-22 11:16:53,843] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50235 )
  INFO root [0x00002448] [2018-07-22 11:16:53,843] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50235, SOMAXCONN=128.
  INFO root [0x00002448] [2018-07-22 11:16:53,844] - server-configs:
  INFO root [0x00002448] [2018-07-22 11:16:53,844] - 	gameUpdateHertz : 10
  INFO root [0x00002448] [2018-07-22 11:16:53,844] - 	entryScriptFile : kbemain
  INFO root [0x00002448] [2018-07-22 11:16:53,844] - 	internalAddr : 192.168.0.106:50235
  INFO root [0x00002448] [2018-07-22 11:16:53,844] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00002448] [2018-07-22 11:16:53,844] - 	componentID : 7001
  WARN root [0x00002448] [2018-07-22 11:16:54,349] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20049, errorno=10048
  INFO root [0x00002448] [2018-07-22 11:16:54,353] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 8508. PID: 6524 ----
  INFO root [0x00002448] [2018-07-22 11:16:54,353] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00002448] [2018-07-22 11:16:54,353] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00002448] [2018-07-22 11:16:54,353] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00002448] [2018-07-22 11:16:54,478] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002448] [2018-07-22 11:16:54,478] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00002448] [2018-07-22 11:16:54,479] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00002448] [2018-07-22 11:16:54,482] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002448] [2018-07-22 11:16:54,482] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00002448] [2018-07-22 11:16:54,482] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002448] [2018-07-22 11:16:54,483] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002448] [2018-07-22 11:16:54,509] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00002448] [2018-07-22 11:16:54,536] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00002448] [2018-07-22 11:16:54,536] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00002448] [2018-07-22 11:16:54,552] - loaded entity-script:Account(1).
 DEBUG root [0x00002448] [2018-07-22 11:16:54,553] - loaded entity-script:CheckName(2).
 DEBUG root [0x00002448] [2018-07-22 11:16:54,558] - loaded entity-script:Halls(3).
 DEBUG root [0x00002448] [2018-07-22 11:16:54,559] - loaded entity-script:Room(4).
  INFO root [0x00002448] [2018-07-22 11:16:54,560] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00002448] [2018-07-22 11:16:54,560] - TelnetServer server is running on port 40000
  INFO root [0x00002448] [2018-07-22 11:16:54,844] - ---- baseapp is running ----
 DEBUG root [0x00002448] [2018-07-22 11:16:54,845] - Components::process(): Request for the process of identity...
  WARN root [0x00002448] [2018-07-22 11:17:00,931] - EntityApp::checkTickPeriod: tick took 6086% (6.09 seconds)!
  INFO root [0x00002448] [2018-07-22 11:17:02,040] - Components::findComponents: find logger(1)...
  INFO root [0x00002448] [2018-07-22 11:17:02,062] - Components::findComponents: found logger, addr:192.168.0.106:50229
  INFO root [0x00002448] [2018-07-22 11:17:02,062] - Components::addComponent: reset orderLog, uid=8508!
  INFO root [0x00002448] [2018-07-22 11:17:02,063] - Components::addComponent[logger], uid=8508, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002448] [2018-07-22 11:17:02,065] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00002448] [2018-07-22 11:17:02,167] - Components::findComponents: found dbmgr, addr:192.168.0.106:50230
  INFO root [0x00002448] [2018-07-22 11:17:02,168] - Components::addComponent: reset orderLog, uid=8508!
  INFO root [0x00002448] [2018-07-22 11:17:02,168] - Components::addComponent[dbmgr], uid=8508, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00002448] [2018-07-22 11:17:02,169] - Components::findComponents: found dbmgr, addr:192.168.0.106:50230
  WARN root [0x00002448] [2018-07-22 11:17:03,169] - Forwarding logs to logger[192.168.0.106:50229/0/0/0]...
 ERROR root [0x00002448] [2018-07-22 11:27:41,832] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00002448] [2018-07-22 11:27:41,837] - The following logs sent to logger failed:
  WARN root [0x00002448] [2018-07-22 11:27:41,837] - ==> [2018-07-22 11:27:41 831] TCPPacketReceiver::processPendingEvents(192.168.0.106:50229): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002448] [2018-07-22 11:27:41,837] - ==> [2018-07-22 11:27:41 832] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00002448] [2018-07-22 11:27:41,883] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50230): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00002448] [2018-07-22 11:27:41,884] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00002448] [2018-07-22 11:27:41,887] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00002448] [2018-07-22 11:27:41,898] - onBaseAppShutDown: state=0
  INFO root [0x00002448] [2018-07-22 11:27:41,898] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00002448] [2018-07-22 11:27:41,898] - onBaseAppShutDown: state=1
  INFO root [0x00002448] [2018-07-22 11:27:41,898] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00002448] [2018-07-22 11:27:41,898] - onBaseAppShutDown: state=2
 ERROR root [0x00002448] [2018-07-22 11:27:41,907] - Account::onCellWriteToDBCompleted(2002): not found dbmgr!
 ERROR root [0x00002448] [2018-07-22 11:27:41,908] - Entity::onDestroy: not found dbmgr!
 ERROR root [0x00002448] [2018-07-22 11:27:41,908] - Account::onCellWriteToDBCompleted(2004): not found dbmgr!
 ERROR root [0x00002448] [2018-07-22 11:27:41,909] - Entity::onDestroy: not found dbmgr!
 DEBUG root [0x00002448] [2018-07-22 11:27:41,909] - Account::~Account(): 2002
 DEBUG root [0x00002448] [2018-07-22 11:27:41,909] - Account::~Account(): 2004
 DEBUG root [0x00002448] [2018-07-22 11:27:41,909] - Room::~Room(): 2003
 DEBUG root [0x00002448] [2018-07-22 11:27:41,910] - Halls::~Halls(): 2001
 DEBUG root [0x00002448] [2018-07-22 11:27:41,910] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00002448] [2018-07-22 11:27:41,910] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00002448] [2018-07-22 11:27:41,910] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00002448] [2018-07-22 11:27:41,911] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00002448] [2018-07-22 11:27:41,911] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00002448] [2018-07-22 11:27:41,911] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00002448] [2018-07-22 11:27:41,911] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00002448] [2018-07-22 11:27:41,911] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00002448] [2018-07-22 11:27:41,920] - Script::uninstall(): is successfully!
 DEBUG root [0x00002448] [2018-07-22 11:27:41,921] - ThreadPool::destroy(): starting size 2.
  INFO root [0x0000079c] [2018-07-22 11:27:42,994] - 
  INFO root [0x0000079c] [2018-07-22 11:27:42,995] - -----------------------------------------------------------------------------------------


  INFO root [0x0000079c] [2018-07-22 11:27:42,995] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x0000079c] [2018-07-22 11:27:42,995] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x0000079c] [2018-07-22 11:27:42,995] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x0000079c] [2018-07-22 11:27:42,999] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x0000079c] [2018-07-22 11:27:42,999] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x0000079c] [2018-07-22 11:27:43,000] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50342 )
  INFO root [0x0000079c] [2018-07-22 11:27:43,000] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50342, SOMAXCONN=128.
  INFO root [0x0000079c] [2018-07-22 11:27:43,001] - server-configs:
  INFO root [0x0000079c] [2018-07-22 11:27:43,001] - 	gameUpdateHertz : 10
  INFO root [0x0000079c] [2018-07-22 11:27:43,001] - 	entryScriptFile : kbemain
  INFO root [0x0000079c] [2018-07-22 11:27:43,001] - 	internalAddr : 192.168.0.106:50342
  INFO root [0x0000079c] [2018-07-22 11:27:43,001] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x0000079c] [2018-07-22 11:27:43,001] - 	componentID : 7001
  INFO root [0x0000079c] [2018-07-22 11:27:44,684] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 10627. PID: 7420 ----
  INFO root [0x0000079c] [2018-07-22 11:27:44,685] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x0000079c] [2018-07-22 11:27:44,685] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x0000079c] [2018-07-22 11:27:44,685] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,809] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,809] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,810] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,813] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,813] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,813] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,814] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,845] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x0000079c] [2018-07-22 11:27:44,882] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x0000079c] [2018-07-22 11:27:44,882] - Script::install(): KBExtra is successfully!
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,900] - loaded entity-script:Account(1).
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,901] - loaded entity-script:CheckName(2).
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,908] - loaded entity-script:Halls(3).
 DEBUG root [0x0000079c] [2018-07-22 11:27:44,909] - loaded entity-script:Room(4).
  INFO root [0x0000079c] [2018-07-22 11:27:44,910] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x0000079c] [2018-07-22 11:27:44,910] - TelnetServer server is running on port 40000
  INFO root [0x0000079c] [2018-07-22 11:27:45,214] - ---- baseapp is running ----
 DEBUG root [0x0000079c] [2018-07-22 11:27:45,214] - Components::process(): Request for the process of identity...
  WARN root [0x0000079c] [2018-07-22 11:27:51,307] - EntityApp::checkTickPeriod: tick took 6093% (6.09 seconds)!
  INFO root [0x0000079c] [2018-07-22 11:27:52,397] - Components::findComponents: find logger(1)...
  INFO root [0x0000079c] [2018-07-22 11:27:52,469] - Components::findComponents: found logger, addr:192.168.0.106:50338
  INFO root [0x0000079c] [2018-07-22 11:27:52,470] - Components::addComponent: reset orderLog, uid=10627!
  INFO root [0x0000079c] [2018-07-22 11:27:52,470] - Components::addComponent[logger], uid=10627, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000079c] [2018-07-22 11:27:52,472] - Components::findComponents: find dbmgr(2)...
  INFO root [0x0000079c] [2018-07-22 11:27:52,574] - Components::findComponents: found dbmgr, addr:192.168.0.106:50339
  INFO root [0x0000079c] [2018-07-22 11:27:52,575] - Components::addComponent: reset orderLog, uid=10627!
  INFO root [0x0000079c] [2018-07-22 11:27:52,575] - Components::addComponent[dbmgr], uid=10627, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x0000079c] [2018-07-22 11:27:52,576] - Components::findComponents: found dbmgr, addr:192.168.0.106:50339
  WARN root [0x0000079c] [2018-07-22 11:27:53,576] - Forwarding logs to logger[192.168.0.106:50338/0/0/0]...
 ERROR root [0x0000079c] [2018-07-22 12:54:48,427] - Components::removeComponentByChannel: cellapp : 8001, Abnormal exit! errorno=10054
